packages/styles/scss/components/progress-indicator/_vars.scss

```diff
@@ -0,0 +1,13 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@use '../../theme' as *;
+
+/// @type Value
+/// @access public
+/// @group progress-indicator
+$progress-indicator-bar-width: 1px inset transparent !default;
```

Comment:
Since there is only one variable, do you think it would make more sense to just have it at the top of the `progress-indicator.scss` file like [Accordion](https://github.com/carbon-design-system/carbon/blob/main/packages/styles/scss/components/accordion/_accordion.scss#L20-L43)? I'm down either way 👍🏻